app/src/main/java/com/vultisig/wallet/presenter/qr_address/QrAddressScreen.kt

```diff
@@ -0,0 +1,135 @@
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.times
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavHostController
+import com.vultisig.wallet.R
+import com.vultisig.wallet.presenter.common.rememberQRBitmapPainter
+import com.vultisig.wallet.ui.components.QrBorder
+import com.vultisig.wallet.ui.theme.Theme
+import com.vultisig.wallet.ui.theme.dimens
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun QrAddressScreen(navController: NavHostController) {
+    val textColor = MaterialTheme.colorScheme.onBackground
+    val appColor = Theme.colors
+    val dimens = MaterialTheme.dimens
+    val viewmodel = hiltViewModel<QrAddressViewModel>()
+
+    Scaffold(
+        topBar = {
+            CenterAlignedTopAppBar(
+                title = {
+                    Text(
+                        text = stringResource(id = R.string.qr_address_screen_title),
+                        style = Theme.montserrat.titleLarge,
+                        fontWeight = FontWeight.Bold,
+                        color = textColor,
+                        modifier = Modifier
+                            .padding(
+                                start = dimens.marginMedium,
+                                end = dimens.marginMedium,
+                            )
+                            .wrapContentHeight(align = Alignment.CenterVertically)
+                    )
+                },
+                colors = TopAppBarDefaults.topAppBarColors(
+                    containerColor = appColor.oxfordBlue800,
+                    titleContentColor = textColor
+                ),
+                navigationIcon = {
+                    IconButton(onClick = {
+                        navController.popBackStack()
+                    }) {
+                        Icon(
+                            imageVector = Icons.AutoMirrored.Filled.KeyboardArrowLeft,
+                            contentDescription = "back", tint = Color.White
+                        )
+                    }
+                },
+            )
+        },
+    ) {
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .consumeWindowInsets(it)
+                .background(Theme.colors.oxfordBlue800),
+            Alignment.Center
+        ) {
+            val screenWidth = LocalConfiguration.current.screenWidthDp
+            val address = viewmodel.address!!
+            val qrBoxSize = ((screenWidth * .8).coerceAtMost(300.0)).dp
+            val imageAndBorderSpace = 30.dp
+            val imageSizeInDp = qrBoxSize - 2 * imageAndBorderSpace
+            val whiteSpace = 10.dp
+
+            Text(
+                text = address,
+                style = Theme.menlo.titleMedium,
```

Comment:
In Figma, and it seems that on iOS it's menlo.body1
```suggestion
                style = Theme.menlo.body1,
```